packages/lesswrong/server/repos/CommentsRepo.ts

```diff
@@ -75,6 +75,57 @@ export default class CommentsRepo extends AbstractRepo<DbComment> {
     `, [limit]);
   }
 
+  readonly bensInterestingDisagreementsCommentId = 'NtsPs9wcwrpeK6KYL';
```

Comment:
This & the associated queries *really* need to be flagged in some way to make sure they don't get used by any forum other than LW.

Reply:
This may be a question of code style, but I'd argue this should really be `private static readonly bensInterestingDisagreementsCommentId = 'NtsPs9wcwrpeK6KYL';` (at the risk of looking like Java).

(This is very much not a full code review - it just caught my eye in a github notification).